src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,86 @@
+.watch {
+  position: relative;
+  top: 100px;
```

Comment:
The watch is not centered vertically. Use flexbox to center the watch both vertically and horizontally.